changelog_unreleased/javascript/pr-7787.md

```diff
@@ -0,0 +1,14 @@
+#### JS: Keep comments position for LogicalExpression ([#7787](https://github.com/prettier/prettier/pull/7787) by [@shisama](https://github.com/shisama))
```

Comment:
This `JS:` should be a error caught by `lint:changelog`, can you update this code

https://github.com/prettier/prettier/blob/aacb5fa75f32c9d596a4e50d122914f14b87c9b2/scripts/lint-changelog.js#L113

Maybe like

```js
if ([...CHANGELOG_CATEGORIES, 'js'].includes(categoryInTitle.toLowerCase())) {
```

to forbid title startsWith `JS:`?